src/universal_transfer_operator/data_providers/database/google/bigquery.py

```diff
@@ -24,15 +72,50 @@ class BigqueryDataProvider(DatabaseDataProvider):
     illegal_column_name_chars_replacement: list[str] = ["_"]
 
     _create_schema_statement: str = "CREATE SCHEMA IF NOT EXISTS {} OPTIONS (location='{}')"
+    NATIVE_PATHS = {
+        FileLocation.GS: "load_gs_file_to_table",
```

Comment:
Looks like we have GS in some places and GCS in some other places. Can we have a consistent naming for this?

Reply:
I have kept it to `gs` everywhere now